Sources/BraintreePayPal/BTContactInformation.swift

```diff
@@ -0,0 +1,14 @@
+/// Contact information of the recipient for the order
+public struct BTContactInformation {
+
+    /// Email address of the recipient
+    let recipientEmail: String?
+    
+    /// Phone number of the recipient
+    let recipientPhoneNumber: BTPayPalPhoneNumber?
+
+    public init(recipientEmail: String? = nil, recipientPhoneNumber: BTPayPalPhoneNumber? = nil) {
```

Comment:
Lets add docstrings here instead of on the internal properties.

Reply:
Sure, updated: [fcd5c15](https://github.com/braintree/braintree_ios/pull/1469/commits/fcd5c1588ee05206423227bb217b3990d6e4a310)